public/js/src/module/records-queue.js

```diff
@@ -127,6 +140,41 @@ function removeAutoSavedRecord() {
     // do not update recordList
 }
 
+/**
+ * Obtains auto-saved record key
+ */
+ function getLastSavedKey() {
+    return lastSavedKey;
+}
+
+/**
+ * @return { Promise<Record | undefined> } a Promise that resolves with a record object or undefined
+ */
+function getLastSavedRecord() {
+    return get( lastSavedKey );
+}
+
+/**
+ * Sets the last-saved record for the provided record's instance.
```

Comment:
I hemmed and hawed on this comment for a while, because the naming makes the accurate wording "save the last saved record as the last saved record", which doesn't really clear anything up.

Reply:
Hehe. Fair enough, maybe just "Sets the last saved record" or no comment at all. What I find difficult about how it's written now is that it's not clear to me what "the provided record's instance" means. I think instance and record are generally synonymous.